protocol/messenger_communities.go

```diff
@@ -2043,6 +2044,57 @@ func (m *Messenger) SetCommunityShard(request *requests.SetCommunityShard) (*Mes
 	return response, nil
 }
 
+func (m *Messenger) SetCommunityMailServers(request *requests.SetCommunityMailServers) (*MessengerResponse, error) {
```

Comment:
Probably makes sense to sync up the community mailservers by calling `func (m *Messenger) syncCommunity`. That function will need to be modified to take into account the community storenodes